background.js

```diff
@@ -32,8 +32,92 @@ function uninstallBlocker() {
 }
 function onBeforeRequest(aDetails) {
   log('onBeforeRequest', aDetails);
-  launch(aDetails.url);
-  return { cancel: true };
+  var list = configs.exclusionielist.trim().split(/\s+/).filter((aItem) => !!aItem);
```

Comment:
This logic should be placed at initialization, instead of this event handler. I think we should not do this for every HTTP requests.